packages/server/build.gradle

```diff
@@ -111,14 +111,6 @@ tasks.named('test') {
     useJUnitPlatform()
 }
 
-hibernate {
-    enhancement {
-        lazyInitialization true
```

Comment:
So this should affect `@OneToMany` annotations and the like - maybe is the same as `@OneToMany(fetch = FetchType.LAZY)`... So all parent child object definitions

Reply:
Yeah, we explicitly use  `FetchType.EAGER` for everything which overrides this, so removing this doesn't do anything.